backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -16,9 +15,13 @@
 import java.util.Set;
 
 @Entity
-@Table(name = "cars")
-@Getter
 @NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@EqualsAndHashCode
+@Builder
```

Comment:
To samo

Reply:
To nie zadziała z Builderem, zostawiłem jak było